tests/integration_tests/build/test_coverage.py

```diff
@@ -116,26 +116,26 @@ def test_coverage(test_fc_session_root_path, test_session_tmp_path):
     print("Thus, coverage is: {:.2f}%".format(coverage))
 
     coverage_low_msg = (
-        'Current code coverage ({:.2f}%) is below the target ({}%).'
-        .format(coverage, coverage_target_pct)
+        "Current code coverage ({:.2f}%) is >{:.2f}% below the target ({}%).".format(
+            coverage, COVERAGE_MAX_DELTA, coverage_target_pct
+        )
     )
 
-    min_coverage = coverage_target_pct - COVERAGE_MAX_DELTA
-    assert coverage >= min_coverage, coverage_low_msg
+    assert coverage >= coverage_target_pct - COVERAGE_MAX_DELTA, coverage_low_msg
 
     # Get the name of the variable that needs updating.
     namespace = globals()
-    cov_target_name = [name for name in namespace if namespace[name]
-                       is COVERAGE_DICT][0]
+    cov_target_name = [name for name in namespace if namespace[name] is COVERAGE_DICT][
+        0
+    ]
```

Comment:
I am not loving this reformatting.

Reply:
Adjusted this [here](https://github.com/firecracker-microvm/firecracker/pull/3021/commits/06160dfda2d6097760f3e5ee56a5f63f86863a01#diff-58497b264df840dfe4603fa26477d633d32c0f2173bf4525265afb814b3d30eaR130-R131). Opinion?

Reply:
Looks good!